src/manifest.json

```diff
@@ -32,7 +32,10 @@
     "128": "icons/icon128.png"
   },
   "web_accessible_resources": ["icons/*.png"],
-  "options_page": "src/options/options.html",
-  "content_security_policy": "script-src 'self' https://cdn.jsdelivr.net 'unsafe-eval'; object-src 'self'",
+  "options_ui": {
```

Comment:
The `options_ui` page is functionally equivalent to `options_page`, but the latter isn’t recognized by Parcel.